test/unittest/StatisticsBackend/CMakeLists.txt

```
@@ -81,13 +93,20 @@ add_executable(calling_user_listeners_tests CallingUserListenersTests.cpp
 target_include_directories(calling_user_listeners_tests PRIVATE
     ${GTEST_INCLUDE_DIRS} ${GMOCK_INCLUDE_DIRS}
     ${PROJECT_SOURCE_DIR}/include
-    ${PROJECT_SOURCE_DIR}/include/${PROJECT_NAME}
+    ${PROJECT_SOURCE_DIR}/include/${PROJECT_NAME_UNDERSCORED}
     ${PROJECT_BINARY_DIR}/include
-    ${PROJECT_BINARY_DIR}/include/${PROJECT_NAME}
+    ${PROJECT_BINARY_DIR}/include/${PROJECT_NAME_UNDERSCORED}
     ${PROJECT_SOURCE_DIR}/src/cpp)
 
 target_link_libraries(calling_user_listeners_tests PUBLIC
-    ${GTEST_LIBRARIES} ${GMOCK_LIBRARIES} fastrtps)
+    ${GTEST_LIBRARIES} ${GMOCK_LIBRARIES} fastrtps fastcdr)
```

Same as before, please check if fastcdr is necessary
```suggestion
    ${GTEST_LIBRARIES} ${GMOCK_LIBRARIES} fastrtps)
```

---

Any test building `types.cxx` will require fastcdr